test/example_app/app/assets/config/manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
```

Comment:
Not needed. Might be a good idea to remove the whole ´app/assets' directory if it does not break the app